kano_settings/system/advanced.py

```diff
@@ -430,7 +430,10 @@ def redirect_traffic_to_localhost():
 
 
 def launch_sentry_server(filename):
-    subprocess.Popen(["sentry -c {}".format(filename)], shell=True)
+    subprocess.Popen(
```

Comment:
I guess one would use it for the convenience of its optional parameters, or run_cmd_log to log if need be. We probably wouldn't have this issue in the first place if we used it though.

Indeed importing the whole of kano utils adds to the lag.